pom.xml

```diff
@@ -46,8 +46,8 @@
     </contributors>
 
     <properties>
-        <jenkins.version>2.89.3</jenkins.version>
-        <crowd-integration-client-rest.version>2.8.8</crowd-integration-client-rest.version>
+        <jenkins.version>2.163</jenkins.version>
+        <crowd-integration-client-rest.version>4.0.2</crowd-integration-client-rest.version>
```

Comment:
I would left current version of crowd for now. This change will only provide a JAVA11 compatibility.
Bumping rest client version would be a breaking change.